pages/docs/concepts/asyncapi-document/server-security.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Server Security 
+weight: 200
+---
+
+Server security refers to the measures and practices implemented to protect servers from unauthorized access, data breaches, and other security threats. Server security involves implementing various security mechanisms to ensure the confidentiality, integrity, and availability of server resources.
+
+In the context of AsyncAPI, securing servers ensures secure exchange of messages between clients and servers. While also protecting sensitive data, preventing unauthorized access, and maintaining the overall security of the API or server.
+
+You can secure a server by using the `security` property where you define or declare which security schemes can be used with the server in context. Each `server` in the AsyncAPI document can have one or more security schemes declared. A security scheme defines a security requirement that must be satisfied to authorize an operation such as a API key or username and password. 
+
+Here is a example of adding security to your server:
+```yml
+asyncapi: '3.0.0'
+info:
+  title: My AsyncAPI
+  version: '1.0.0'
+servers:
+  production:
+    url: api.example.com
+    protocol: amqp
+    security:
+      - saslScram: []
+      - certs: []
+components:
+  securitySchemes:
+    saslScram:
+      type: scramSha256
+      description: Provide your username and password for SASL/SCRAM authentication
+    certs:
+      type: X509
+      description: Download the certificate files from service provider
+```
+
+Here is a illustration of securing servers: 
+```mermaid
+graph LR
+  A[asyncapi: '3.0.0'] --> B[info]
```

Comment:
the approach for the diagram is not consistent with other diagrams from other 3.0 docs, like even yours about tags

- colors do not match
- you provide some additional components, not related, like info, but not sure why and also not as an individual components but as `title, version`

what do you want to represent with this diagram, what is its goal in this document - you need to answer these questions to yourself 😄

Reply:
Simplified and added a straight forward diagram now which is also consistent with the other docs and color scheme. Does this work well or do I need me to make any further changes ?